.github/workflows/cron.yml

```diff
@@ -30,22 +30,22 @@ jobs:
     continue-on-error: true
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v2
```

Comment:
Curious, is there a good reason this set of actions needs to be different from ci-mgmt standard set of actions? Possibly just lack of time to prioritize folding it back?

Reply:
awsx is simply not in ci-mgmt, and I didn't have the time to take that on. Not sure if it's even a great idea because awsx is not a provider in the same sense than the others.

As to the action versions, I simply chose the latest one if they worked.

Reply:
Yeah, it's a provider. It provisions resources, has a schema and publishes to PyPI/NPM. You're right that the tradeoffs need to be carefully considered however I'd not dismiss it out of hand. For a recent example, keeping java versions up to date, migrating to python wheels, these little infra tasks roll trivially through standard providers managed in ci-mgmt but require special work for every provider like this one.